docs/release-notes/3.0.0_Alpha.md

```diff
@@ -0,0 +1,91 @@
+# Percona Monitoring and Management 3.0.0 Alpha
+
+| **Release date** | April 30th, 2024                                                                                   |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+It enables you to observe the health of your database systems, explore new patterns in their behavior, troubleshoot them and execute database management operations—regardless of whether your databases are located on-premises or in the cloud.
+
+## Release highlights
+
+PMM version 3.x marks a new milestone for PMM, now built on a more secure foundation, with an enhanced user experience and easier deployment across a broader range of environments.
+
+## Improved security
+
+### Enhanced API authentication with Grafana service accounts
+
+With the release of PMM 3.x, we've made a significant enhancement to the way API authentication is handled. PMM no longer relies on API keys as the primary method for controlling access to the PMM Server components and resources. Instead, PMM now leverages Grafana service accounts, which provide a more secure and manageable authentication mechanism compared to API keys.
+
+The transition to service accounts brings several advantages:
+
+- Fine-grained access control, ensuring that each account gets access only to the necessary resources.
+- Logging of all actions performed, providing better visibility and auditing.
+- Easy rotation of credentials that doesn't impact associated applications.
+
+When you install PMM 3.x, any existing API keys will be seamlessly converted to service accounts with corresponding service tokens. For more information about using service accounts in PMM, see [Service accounts authentication](../api/authentication.md).
+
+![Service Accounts page](../_images/Service_Accounts.png)
+
+### More options for deploying PMM rootless
+
+In addition to Podman, PMM Server can also now be deployed rootless with Helm, Docker, Virtual Appliance, or Amazon AWS.
+
+This setup prioritizes security by eliminating the need for root privileges to create, run, and manage containers. Running PMM Server as a non-root user means that you never have to give a user root permission on the host. This adds an extra layer of security, essential for protecting against potential security breaches.
+
+For detailed instructions on deploying rootless PMM, check the [Setting up PMM Server](https://docs.percona.com/percona-monitoring-and-management/setting-up/index.html#set-up-pmm-server) topic.
+
+### Transition to Oracle Linux 9 completed
```

Comment:
maybe shorten this section to just say - RHEL 7 is dead , so no EL 7 builds, everything on OL9 - as most secured and recent version of OS